build.sh

```diff
@@ -668,6 +669,11 @@ case "$COMMAND" in
         exit 0
         ;;
 
+    "verify-swiftlint")
+        swiftlint lint --strict
+        exit 0
```

Comment:
Actually it's fine, since `set -o pipefail`/`set -e` means that `exit 0` won't be reached if `swiftlint lint --strict` fails.